events/netherlands/balfolk_nl.yaml

```diff
@@ -91,6 +92,23 @@ events:
     social: true
     price: donation
 
+  - name: Fest Noz Hoorn
+    links:
+      - "https://www.balfolk.nl/agenda/fest-noz-hoorn/"
+      - "https://www.facebook.com/events/2864813930451126/"
```

Comment:
This looks like the wrong Facebook event, it says "Balfolk Café Nijmegen met Hartwin Dhoore & Geronimo".

Reply:
Was a copy paste error, took the format from another event, fixed in new commit